Actions/ReadSettings/ReadSettings.ps1

```diff
@@ -54,6 +54,11 @@ if ($settings.versioningstrategy -ne -1) {
     }
 }
 
+if ($settings.buildEnvironmentName -and -not $settings.useCompilerFolder) {
+    Write-Host "Using an online environment for testing, use CompilerFolder to build the app"
```

Comment:
Suggestion; Make it a warning? This indicates to me that they repo owner has misconfigured something in their AL-Go settings. We catch it so this workflow doesn't fail but they should fix their settings.

Reply:
Actually, the idea here is that people shouldn't care - if they specify that they want to use an online environment for testing, then we automatically find out that using compilerfolder is the best option.